build.gradle

```diff
@@ -8,15 +8,15 @@ buildscript {
     dependencies {
         classpath 'com.gradle.publish:plugin-publish-plugin:0.9.1'
         classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.6'
-        classpath 'com.netflix.nebula:nebula-dependency-recommender:4.0.2'
-        classpath 'com.netflix.nebula:nebula-publishing-plugin:4.9.1'
+        classpath 'com.netflix.nebula:nebula-dependency-recommender:4.2.1'
+        classpath 'com.netflix.nebula:nebula-publishing-plugin:5.1.0'
```

Comment:
Checked on local publish. I bumped these since without the bump versions were missing